packages/headless/src/features/insight-search/insight-search-analytics-actions.ts

```diff
@@ -41,19 +41,17 @@ export const logContextChanged = (
     client.logContextChanged(meta);
   });
 
-export const logExpandToFullUI = (
-  caseId: string,
-  caseNumber: string,
-  fullSearchComponentName: string,
-  triggeredBy: string
-): InsightAction =>
-  makeInsightAnalyticsAction('analytics/expandToFullUI', (client, state) => {
-    const meta = {
-      caseId,
-      caseNumber,
-      fullSearchComponentName,
-      triggeredBy,
-      caseContext: state.insightCaseContext?.caseContext || {},
-    };
-    client.logExpandToFullUI(meta);
+export const logInsightInterfaceLoad = (): InsightAction =>
+  makeInsightAnalyticsAction('analytics/interface/load', (client, state) =>
+    client.logInterfaceLoad(
+      getCaseContextAnalyticsMetadata(state.insightCaseContext)
+    )
+  );
+
+export const logInsightInterfaceChange = (): InsightAction =>
+  makeInsightAnalyticsAction('analytics/interface/change', (client, state) => {
+    client.logInterfaceChange({
+      ...getCaseContextAnalyticsMetadata(state.insightCaseContext),
+      interfaceChangeTo: state.configuration.analytics.originLevel2,
```

Comment:
I'm assuming there's no breaking changes to any of this?

Reply:
Yeah just changed the where the actions is defined but everything is exported and exposed like it was before so no breaking changes, just a slightly better organization.